.editorconfig

```diff
@@ -0,0 +1,8 @@
+[*]
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{y,ya}ml]
+indent_size = 2
```

Comment:
Perhaps add `.editorconfig` to `.gitignore` so that it isn't included in the PR?

Reply:
Also, this project uses an indent_size of 2 for shell scripts.

Reply:
I added it on purpose, since with multiple people working on this project I saw a few inconsistenties. But if you want I can remove it from the PR and add it to the .gitignore

Reply:
With the indent size of 2: in the changes I did, I had a few indents of 2 to change to 4 and then all was indented with 4. But it's not a big deal to go from 4 to 2. It's your call, you are the owner :)

Reply:
You make a good point. Without clear guidelines, it makes it difficult for us to be consistent. My preference is indentation of 2 as I've often found the code is far more prone to wrapping with 4 - and that 2 provides enough of a visual indicator to clearly see indentation.

I wasn't familiar with `.editorconfig` and thought it was specific to an editor, now that I've looked it up and see it's for all editors - I like the idea. Let's keep it!

Reply:
Changed it to indent 2, and fixed it in all scripts.